core/src/main/kotlin/materialui/components/breadcrumbs/breadcrumbs.kt

```diff
@@ -0,0 +1,32 @@
+package materialui.components.breadcrumbs
+
+import kotlinx.html.DIV
+import kotlinx.html.Tag
+import kotlinx.html.TagConsumer
+import materialui.components.StandardProps
+import materialui.components.toolbar.enums.ToolbarStyle
+import react.*
+
+@JsModule("@material-ui/core/Breadcrumbs")
+private external val breadcrumbsModule: dynamic
+
+external interface BreadcrumbsProps : StandardProps {
+    val separator: String?
```

Comment:
`separator` prop type is node. In the case of node type, please declare it as dynamic or ReactElement for now

If you want to make it easier to assign values such as String, please define an appropriate extended function.

https://material-ui.com/api/breadcrumbs/#props